src/Microsoft.Diagnostics.Monitoring.WebApi/Strings.resx

```diff
@@ -121,6 +121,10 @@
     <value>Unable to get process environment.</value>
     <comment>Gets a string similar to "Unable to get process environment.".</comment>
   </data>
+  <data name="ErrorMessage_HttpEgressDisabled" xml:space="preserve">
+    <value>Http Egress was not enabled.</value>
```

Comment:
```suggestion
    <value>HTTP egress is not enabled.</value>
```

Reply:
Just so I know the convention, do we use Http in the context of a variable and HTTP in the context of documentation? Just so you don't need to keep manually fixing them all for me :)

Reply:
Yep, when written for consumption via reading, it is "HTTP". When use in code, it is "Http". By convention, (if I recall the exact number), if an acronym is more than 3 letters, then the first is uppercased while the rest are lowercased (unless its the first word or a field of local variable, which then it is all lowercase).